File: Microsoft.Recognizers.Text.NumberWithUnit/English/Parsers/TemperatureParserConfiguration.cs
using Microsoft.Recognizers.Text.NumberWithUnit.English.Extractors;
using Microsoft.Recognizers.Text.Number.Utilities;
using System.Globalization;

namespace Microsoft.Recognizers.Text.NumberWithUnit.English.Parsers
{
    public class TemperatureParserConfiguration : EnglishNumberWithUnitParserConfiguration
    {
        public TemperatureParserConfiguration() : this(new CultureInfo(Culture.English)) { }

        public TemperatureParserConfiguration(CultureInfo ci) : base(ci)
        {
            this.BindDictionary(TemperatureExtractorConfiguration.TemperatureSuffixList);
        }
    }
}

File: Java/libraries/recognizers-text-number-with-unit/src/main/java/com/microsoft/recognizers/text/numberwithunit/english/parsers/TemperatureParserConfiguration.java
package com.microsoft.recognizers.text.numberwithunit.english.parsers;

import com.microsoft.recognizers.text.Culture;
import com.microsoft.recognizers.text.CultureInfo;
import com.microsoft.recognizers.text.numberwithunit.english.extractors.TemperatureExtractorConfiguration;

public class TemperatureParserConfiguration extends EnglishNumberWithUnitParserConfiguration {

    public TemperatureParserConfiguration() {
        this(new CultureInfo(Culture.English));
    }

    public TemperatureParserConfiguration(CultureInfo cultureInfo) {
        super(cultureInfo);

        this.bindDictionary(TemperatureExtractorConfiguration.TemperatureSuffixList);
    }
}
